docs/SETUP_CI_CD_PIPELINE.md

```diff
@@ -161,34 +249,114 @@ They have two very important use:
 us to store variables and specify for which environment this variable is bound to (for organisation and security).
  * Gitlab stages and jobs must be tied to a specific environment, so that pipelines don't leak variables.
 
+#### Create DNS record for accessing endpoint on staging and on live servers
```

Comment:
Maybe this section would be better placed after the `AWS dashboard` section because DNS A records cannot be created with knowing the elastic IP addresses?